### PR TITLE
Update JS Docs

### PR DESCRIPTION
**Pull request by juan-fernandez**

### CODE REVIEW

File: docs/nodejs-compatibility.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ The instrumentation is automatic.
 | -------------------------- | :-----------------: | :----: | :-----: | :--: |
 | [Jest](https://jestjs.io/) |          ✓          |   ✓    |    ✓    |  ✓   |
```
Comment (fermayo) [inline docs/nodejs-compatibility.md]: Remove the HTTP column here

File: docs/nodejs-compatibility.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ The instrumentation is automatic.
...
 
+### HTTP libraries
+
+All http libraries that rely on [Node.js http](https://nodejs.org/api/http.html) are automatically instrumented.
```
Comment (fermayo) [inline docs/nodejs-compatibility.md]: This should be a table, like https://docs.scope.dev/docs/go-compatibility#libraries
Native libraries show like third party libraries.

File: docs/nodejs-compatibility.md [hunk 3]
```diff
@@ -12,6 +12,10 @@ The instrumentation is automatic.
 | -------------------------- | :-----------------: | :----: | :-----: | :--: |
 | [Jest](https://jestjs.io/) |          ✓          |   ✓    |    ✓    |  ✓   |
 
+### HTTP libraries
```
Comment (fermayo) [inline docs/nodejs-compatibility.md]:
```suggestion
### Libraries
```

File: docs/nodejs-compatibility.md (hunk 2 above)
Comment (fermayo) [reply docs/nodejs-compatibility.md]: Also, is that the client, or the server, or both?

Comment (juan-fernandez) [reply docs/nodejs-compatibility.md]: both

Comment (juan-fernandez) [reply docs/nodejs-compatibility.md]: well, actually both in testing. In runtime only server.

**Pull request merged**